### PR TITLE
Add support for content syncing

### PR DESCRIPTION
**Pull request by donnapep**
- Add support for content syncing such that any item that does not report as ready will have its duration added to the next playlist item following it that _is_ ready.
- Move tests from `rise-playlist-duration.html` and `rise-playlist-not-ready.html` to `rise-playlist-content-sync.html`.
- Rename `e2e` folder to `integration`.

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees Please review. Thx.

File: README.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ var readyEvent = new CustomEvent("rise-component-ready", {
 this.dispatchEvent(readyEvent);
 ```
 
+The component has 30 seconds in which to initialize and dispatch the `rise-component-ready` event. After that, only components that have reported as ready will display. Components that are skipped will have their playlist item duration (if specified) added to the duration of the next playlist item following it that reported as being ready.
```
Comment (stulees) [inline README.md]: @donnapep The last sentence _"Components that are skipped will have their playlist item duration (if specified) added to the duration of the next playlist item following it that reported as being ready."_ doesn't read well at the end. I think it's the use of "that". Can you adjust?

**Comment by stulees** [comment]
@donnapep One suggestion, otherwise :+1:

**Comment by donnapep** [comment]
@stulees Adjusted the text. Does it read better now?

**Comment by stulees** [comment]
@donnapep Perfect

**Pull request merged**